charts/studio/templates/configmap-studio.yaml

```
@@ -121,6 +121,9 @@ data:
   DQL_UDF_ENABLED: {{ $dvcx.udfEnabled | default "False" | quote }}
   DVCX_ENABLED: {{ $dvcx.enabled | default "False" | quote }}
   DVCX_UDF_ENABLED: {{ $dvcx.udfEnabled | default "False" | quote }}
+  {{- if .Values.ray.enabled }}
```

```suggestion
  {{- if (.Values.ray).enabled }}
```

Should work - https://stackoverflow.com/questions/59795596/how-to-make-nested-variables-optional-in-helm/68807258#68807258

I didn't know that before ;-)

---

How come it works in our other templates? e.g. `{{- if .Values.global.ingress.enabled -}}` in `ingress-studio-api.yaml`

---

Because `values.yaml` on the `main` branch already contained those keys; see https://github.com/iterative/helm-charts/pull/123#issuecomment-1681253297

---

Ah, I see. Thanks!